.prettierrc.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
This shouldn't be needed, as it should come from `package.json#prettier` (which should be set to `prettier-config-ackama`).